config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-end
+
+  get '/users', to: 'users#index'
+  get '/users/745', to: 'users#show'
+  get '/users/745/posts', to: 'posts#index'
+  get '/users/745/posts/3', to: 'posts#show'
```

Comment:
- [ ] It would be nice to dynamically make your routes like the  example below ⬇️, you don't need to specify a specified id 👍🏿 



```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
thank you very much